portfolio/src/main/webapp/script.js

```diff
@@ -26,3 +26,20 @@ function addRandomGreeting() {
   const greetingContainer = document.getElementById('greeting-container');
   greetingContainer.innerText = greeting;
 }
+
+function addRandomFact(){
+    const facts = [
+    'I am currently learning russian,Привет ', 
+    'I am interested in compilers and progamming languages',
+    'I have lived in three states: California, New York, and Virginia !',
+    ];
+
```

Comment:
Can this be a constant defines at the top-level of the file?

Reply:
That makes more sense ! You wouldn't want to have to create the variables after each click